doc/source/developer_guide/dg_tools.rst

```diff
@@ -0,0 +1,150 @@
+The weights file can be re-used if interpolating different data on the same grid. 
+Although not tested in this version of the scripts, additional regridding options 
+are available by xesmf, including ‘conservative’ and ‘nearest neighbor’. These 
+methods have not been tested in the current version of the scripts. The reader 
+is referred to the xESMF web page for further documentation 
+(https://xesmf.readthedocs.io/en/latest/ last accessed 5 NOV 2020).
+
+To use the **interp_jra55_ncdf_bilinear** script, do ::
+
+  python3 interp_jra55_ncdf_bilinear.py –h
+
+to see the latest interface information ::
+
+  usage: interp_jra55_ncdf_bilinear.py [-h] JRADTG gridout ncout
+
+  Interpolate JRA55 data to CICE grid
+
+  positional arguments:
+    JRADTG      JRA55 input file date time group
+    gridout     CICE grid file (NetCDF)
+    ncout       Output NetCDF filename
+
+  optional arguments:
+    -h, --help  show this help message and exit
+
+Sample usage is ::
+
+  ./interp_jra55_ncdf_bilinear.py 1996010100_1996033121 grid_gx3.nc JRA55_gx3_03hr_forcing_1996-q1.nc
+  ./interp_jra55_ncdf_bilinear.py 1996040100_1996063021 grid_gx3.nc JRA55_gx3_03hr_forcing_1996-q2.nc
+  ./interp_jra55_ncdf_bilinear.py 1996070100_1996093021 grid_gx3.nc JRA55_gx3_03hr_forcing_1996-q3.nc
+  ./interp_jra55_ncdf_bilinear.py 1996100100_1996123121 grid_gx3.nc JRA55_gx3_03hr_forcing_1996-q4.nc
+
+In this case, the 4 quarters of 1996 JRA55 data is going to be interpolated to the gx3 grid.
+NCO can be used to aggregate these files into a single file ::
+
+  ncrcat JRA55_gx3_03hr_forcing_1996-??.nc JRA55_${grid}_03hr_forcing_1996.nc
+
+NOTES
+
+- The scripts are designed to read a CICE grid file in netCDF format.  This is the "grid_gx3.nc" file above.  The NetCDF grid names are hardcoded in **interp_jra55_ncdf_bilinear.py**. If you are using a different grid file with different variable names, this subroutine needs to be updated. 
+- All files should be placed in a common directory.  This includes the raw JRA55 input files, the CICE grid file, and **interp_jra55_ncdf_bilinear.py**.  The output files will be written to the same directory.
+- The script **configuration/tools/jra55_datasets/make_forcing.csh** was used on the NCAR cheyenne machine in March, 2021 to generate CICE forcing data.  It assumes the raw JRA55 is downloaded, but then sets up the python environment, links all the data in a common directory, runs **interp_jra55_ncdf_bilinear.py** and then aggregates the quarterly data using NCO.
+- The new forcing files can then be defined in the **ice_in** namelist file using the input variables, ``atm_data_type``, ``atm_data_format``, ``atm_data_dir``, ``fyear_init``, and ``ycycle``.  See :ref:`forcing` for more information.
+- The total precipitation field is mm/day in JRA55.  This field is initially read in as snow, but prepare_forcing in **ice_forcing.F90** splits that into rain or snow forcing depending on the air temperature.
```

Comment:
Technically, it's probably mm/day of liquid water (rain), which is read into a variable called 'fsnow' before being adjusted with snow density in the forcing module.  But that's a minor quibble.

Reply:
It looks like both snow and rain are in units of kg/m2-s in the forcing.  I assume that water flux is converted to a snow depth output the forcing module?